README.md

```diff
@@ -95,40 +80,60 @@ Moreover, it provides a dashboard that helps you to analyze the data that your c
 
 The frontend site uses HTTPs, in order to have it communicate with the processor, the latter must be either **Running with SSL enabled** (See [Enabling SSL](#enabling-ssl) on this Readme), **or** you must edit your site settings for `https://beta.lanthorn.ai` in order to allow for Mixed Content (Insecure Content). **Without doing any of these, communication with the local processor will fail**
 
-#### Run the processor
+#### Running the processor
 
 Make sure you have `Docker` installed on your device by following [these instructions](https://docs.docker.com/install/linux/docker-ce/debian).
 The command that you need to execute will depend on the chosen device because each one has an independent Dockerfile.
 
-##### Run on Jetson Nano
+There are two alternatives to run the processor in your device:
+  1. Using `git` and building yourself the docker image.
+  2. Pulling the image (already built) from [neuralet Docker Hub repository](https://hub.docker.com/repository/docker/neuralet/smart-social-distancing).
```

Comment:
I would add something like: "Follow the guide in this section" with a link to the section in the Readme

Reply:
Added